docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -11,61 +11,72 @@
     <dependency>
       <groupId>com.azure.spring</groupId>
       <artifactId>spring-cloud-azure-dependencies</artifactId>
-      <version>version</version>
+      <version>version</version> <!--1-->
       <type>pom</type>
       <scope>import</scope>
     </dependency>
   </dependencies>
 </dependencyManagement>
 ----
-
-The version for spring-cloud-azure-dependencies is {project-version}.
+<1> The version for spring-cloud-azure-dependencies is {project-version}.
 
 [#starter-dependencies]
 ==== Starter Dependencies
 
-Spring Cloud Azure Starters are a set of convenient dependency descriptors to include in your application.Each starter contains all the dependencies and transitive dependencies needed to begin using their corresponding Spring Cloud Azure module.It boosts your Spring Boot application development with Azure services.
+Spring Cloud Azure Starters are a set of convenient dependency descriptors to include in your application. Each starter includes all the dependencies and transitive dependencies needed to begin using their corresponding Spring Cloud Azure modules. They boost your Spring Boot application development with Azure services.
```

Comment:
`dependency descriptors to include in your application` can be dependency descriptors to `be` include in your application.

their corresponding Spring Cloud Azure modules  ->
its corresponding Spring Cloud Azure module

Reply:
I think this is more like `dependency descriptors (for you) to include in your application`